pkg/settings/settings.go

```diff
@@ -308,6 +311,37 @@ func (settings *Settings) ShouldInitialism() bool {
 	return !settings.NoInitialism
 }
 
+// HasCustomRename returns if the setting to rename columns with
+// custom rename rules is set.
+func (settings *Settings) HasCustomRename() bool {
+	return settings.CustomColumnRename != ""
+}
+
+const multiSplitter = ","
+const ruleSplitter = ":"
```

Comment:
I like but can also hard-code where it's used. If you decide to keep them here, what do you think about to rename and grouping these like

```go
const (
	rulesSeparator = ","
	ruleSeparator  = ":"
)
```